CMakeLists.txt

```diff
@@ -50,11 +49,17 @@ set_package_properties(ZLIB PROPERTIES
     TYPE REQUIRED
 )
 
+
+find_package(GLM REQUIRED)
+find_package(GLEW REQUIRED)
```

Comment:
GLEW is missing the extra URL and Description fields.

https://github.com/nigels-com/glew and `The OpenGL Extension Wrangler Library` are sufficient.